### ISSUE TITLE
Compare new (infected) keys more often than 24 hours to reduce latency

### ISSUE THREAD

**Issue by bufemc** [corona-warn-app/cwa-app-android#1380]
**Current Implementation**
It seems infected keys are retrieved/downloaded and compared only once every 24 hours.

**Suggested Enhancement**
Download and compare keys more often, e.g. four times a day.

**Expected Benefits**
User is informed (and can self-quarantine) earlier, latency and risk to spread a possible infection is reduced.

**Explanation**
I think that in view of the increasing number of infections, obtaining new data only once every 24 hours is an unnecessary latency which could be reduced. Worst case scenario in my head: let's say the server has new data ("infected keys") at 1:00 o'clock in the morning, but my mobile phone updated at 23:00 (11 pm) o'clock the day before, so it is somehow outdated. Imagine you get in contact with one of these new infected persons registered on the server. Then I have to wait until 11pm again to get informed. Right, wrong? If right, I would propose to update at least e.g. four times a day, to reduce the IMHO unnecessary latency.

I am using both OS, do I have to clone/create this ticket on iOS, too? What is the reason to obtain infected keys only once in 24 hours? Server load? But I will wait for comments first.. maybe I am totally wrong?

If I am somehow wrong please explain why, and then it's okay for me to close the ticket.

Benefit: get informed earlier, instead of spreading an infection up to 1 day.

**Comment by daimpi** [comment]
Hi @bufemc,

I fully agree with the idea for higher frequency checks. This issue is already tracked over here: https://github.com/corona-warn-app/cwa-backlog/issues/2

I think you can close this one here and continue the discussion over there 🙂. 

Regarding
> I am using both OS, do I have to clone/create this ticket on iOS, too?

Usually for issues affecting both iOS and Android you'd just open one issue either in the documentation repo (for problems/bugs) or in the wishlist repo (for enhancement requests/ideas) afaik.

**Comment by Ein-Tim** [comment]
@daimpi 
Correct me if I'm wrong here but afaik the downloaded keys should be compared all 2 hours, but new keys are only downloaded all 24h...

**Comment by heinezen** [comment]
Hey @bufemc ,

What you describe is already on the developer's To Do list. As @daimpi already pointed out, progress is tracked in https://github.com/corona-warn-app/cwa-backlog/issues/2 . You can subscribe to the thread there to get notified about updates. This issue will be closed as a duplicate.

Regards,
CH

----
Corona-Warn-App Open Source Team

**Comment by heinezen** [comment]
Duplicate of https://github.com/corona-warn-app/cwa-backlog/issues/2
